### DISCUSSION TITLE
Problems to get the commits from a repository on two projects

### DISCUSSION THREAD

**Discussion by SAADev03** [vercel/community#1987] [Help]
### Summary

Hi, guys.

I have two projects from the same repository, one on the Hobby plan (that receive all the commits on the "Deployment" panel) and other on the Pro Trial (that don't receive the news commits). I need to get all the commits from this repository in all projects i have.

I tried to delete and create the project again on the Pro Trial Team's view, transfer the project from the Hobby Plan to the Pro Trial but i'm still not getting the commits in all projects.

How could i solve this?

**Comment by amyegan** [comment]
Hi @SAADev03. Are you getting _some_ of the commits, or are none of the commits coming through in the second project?
